shared/chat/conversation/messages/wrapper/send-indicator/index.tsx

```diff
@@ -116,17 +127,29 @@ class SendIndicator extends React.Component<Props, State> {
     this.sentTimeoutID && clearTimeout(this.sentTimeoutID)
   }
 
+  private animationType = () => {
+    let animationType = Styles.isDarkMode()
+      ? statusToIconDark[this.state.animationStatus]
+      : statusToIcon[this.state.animationStatus]
+    // There is no exploding-error state
+    if (this.props.isExploding && this.state.animationStatus !== 'error') {
+      // Don't show the sent state when exploding
+      if (this.state.animationStatus === 'sent') {
+        this.setState({visible: false})
```

Comment:
we should not call setState in a render

Reply:
What's your recommendation?

Reply:
How's this? https://github.com/keybase/client/pull/23464/commits/7d22950232adb309b7a7dc787d136e5d22b503be

Reply:
sure